doc/fastfetch.1.in

```diff
@@ -0,0 +1,146 @@
+.TH FASTFETCH 1 "@FASTFETCH_BUILD_DATE@" "@CMAKE_PROJECT_NAME@ @CMAKE_PROJECT_VERSION@"
+
+.SH NAME
+fastfetch - a neofetch-like tool for fetching system information and
+displaying them in a pretty way
+
+.SH SYNOPSIS
+
+.B fastfetch
+.I [options...]
+
+.SH DESCRIPTION
+
+Fastfetch is a neofetch-like tool for fetching system information and
+displaying them in a pretty way. It is written mainly in C, with performance
+and customizability in mind.
+Currently, Linux, Android, FreeBSD, MacOS and Windows 7+ are supported.
+
+.SH "EXIT STATUS"
+
+On successfull execution, fastfetch returns zero. If any error happened,
+the exit code will be non-zero.
+
+.SH OPTIONS
+
+.SS "Informative Options"
+.TP
+
+.B \-\-help \fI [command]
+Show help output, displaying all available options or help for given command
+.TP
+
+.B \-\-version
+Show version of fastfetch
+.TP
+
+.B \-\-list\-config\-paths
+List search paths of config files
+.TP
+
+.B \-\-list\-data\-paths
+List search paths of presets and logos
+.TP
+
+.B \-\-list\-logos
+List available logos, they can be loaded with \fI \-\-logo
+.TP
+
+.B \-\-list-modules
+List available modules
+.TP
+
+.B \-\-list-presets
+List available presets, they can be loaded with \fI \-\-load\-config
+.TP
+
+.B \-\-list-features
+List the supported features fastfetch was compiled with
+.TP
+
+.B \-\-print-logos
+Print available logos
+.TP
+
+.B \-\-print\-default\-structure
+Print the default structure
+
+
+.SS "Display Options"
+.TP
+
+.B \-l, \-\-logo \fI <logo>
+Set the logo to display
+.TP
+
+.B \-c, \-\-structure \fI <structure>
+Set the structure of the fetch. For details about the structure,
+see the CONFIGURATION section.
+
+
+.SS "Config Options"
+.TP
+
+.B \-\-print\-config\-system
+Print the default system config
+.TP
+
+.B \-\-print\-config\-user
+Print the default user config
+.TP
```

Comment:
These two options no longer works in dev branch, and about to be removed. Let's not add these.

Reply:
Removed in a4c3d600a89a733aea33905c4b124e76fc1e5195

I also removed them from help.txt in e10fb1f17cc216a13be7f0eebb75b65b54aca455